product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.1-converting-soap-to-json/1.1.3-soap-to-json-using-data-mapper/src/test/java/org/wso2/carbon/esb/scenario/test/SoapToJsonUsingDataMapperTest.java

```diff
@@ -48,6 +49,9 @@ public void init() throws Exception {
 
     @Test(description = "1.1.3.1-Valid Soap To Json transformation Using Data Mapper", enabled = true)
     public void convertValidSoapToJsonUsingDataMapper() throws Exception {
+        // WSO2 ESB 4.9.0 does not support Datamapper mediator
+        skipTestsForIncompatibleProductVersions(ScenarioConstants.VERSION_490);
```

Comment:
shall we call skipTestsForIncompatibleProductVersion() in init()?

Reply:
Fixed